app/controllers/readers_controller.rb

```diff
@@ -1,12 +1,26 @@
 class ReadersController < ApplicationController
   def index
-    @readers = Reader.all
-    @sorted_readers = Reader.sort_by_most_recently_created
+    # @readers = Reader.all
+    @readers = Reader.sort_by_most_recently_created
     
   end
 
   def show
     @reader = Reader.find(params[:id])
     @book_number= @reader.books.size
   end
+
+  def new
+  
+  end
+
+  def create
```

Comment:
Create method with strong params, b/c I wasn't sure how to create with the boolean check boxes I made in the form